src/components/Tool/Appeal/List/AppealsListFilterPanel/AppealsListFilterPanelItem.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { ReactElement } from 'react';
+import ArrowForwardIos from '@mui/icons-material/ArrowForwardIos';
+import {
+  Box,
+  ListItem,
+  ListItemText,
+  Skeleton,
+  Typography,
+} from '@mui/material';
+import { makeStyles } from 'tss-react/mui';
+import theme from 'src/theme';
+import { AppealStatusEnum } from '../../AppealsContext/AppealsContext';
+
+const useStyles = makeStyles()(() => ({
+  li: {
+    borderBottom: '1px solid #EBECF1',
+    '&:hover': {
+      backgroundColor: theme.palette.cruGrayLight.main,
+    },
+  },
+}));
+
+const countBoxStyles = (id: AppealStatusEnum): object => {
+  let styles: object = {
```

Comment:
What do you think about making this a `const`, renaming it to `sharedStyles`, and then in the cases, doing `return { ...sharedStyles, backgroundColor: theme.palette.statusDanger.main }`? That way we don't have to mutate `styles` or break out of each case branch (because we're returning).